ray-operator/apis/ray/v1alpha1/raycluster_types.go

```diff
@@ -65,23 +65,28 @@ type ScaleStrategy struct {
 
 // AutoscalerOptions specifies optional configuration for the Ray autoscaler.
 type AutoscalerOptions struct {
-	// Resources specifies resource requests and limits for the autoscaler container.
-	// Default values: 256m CPU request, 512m CPU limit, 256Mi memory request, 512Mi memory limit.
+	// Resources specifies optional resource request and limit overrides for the autoscaler container.
+	// Default values: 500m CPU request and limit. 512Mi memory request and limit.
```

Comment:
Is this per node? This is insanely low, typical Ray nodes should be 8-32GiB in size, or more than 10-40x this value. Ray is not designed to operate with tiny nodes.

Reply:
These are defaults for the autoscaler container, not for a Ray node.

Reply:
Important related discussion:
https://github.com/ray-project/kuberay/issues/417